<!-- 
  Simulate a differential drive robot with the teb_local_planner in stage:
  - stage
  - map_server
  - move_base
  - static map
  - amcl
  - rviz view
 -->
<launch>
  <remap from="map" to="/rtabmap/grid_map"/>
  <remap from="odom" to="/odometry/filtered"/>
  <arg name="spacing" default="0.4" />
        <!--  ************** Navigation ***************  -->
	<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
  	  	<rosparam file="$(find robot_mower_2dnav)/cfg/costmap_common_params.yaml" command="load" ns="global_costmap" />
  	 	<rosparam file="$(find robot_mower_2dnav)/cfg/costmap_common_params.yaml" command="load" ns="local_costmap" />
  		<rosparam file="$(find robot_mower_2dnav)/cfg/local_costmap_params.yaml" command="load" />
  		<rosparam file="$(find robot_mower_2dnav)/cfg/global_costmap_params.yaml" command="load" />
  		<rosparam file="$(find robot_mower_2dnav)/cfg/teb_local_planner_params.yaml" command="load" />

		<param name="base_global_planner" value="carrot_planner/CarrotPlanner" />
		<param name="allow_unknown" value="true" />
		<param name="planner_frequency" value="1.0" />
		<param name="planner_patience" value="5.0" />

		<param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />
		<param name="controller_frequency" value="1.0" />
		<param name="controller_patience" value="15.0" />
	</node>
<<<<<<< HEAD
=======

<node name="twist_rc" pkg="robot_mower_2dnav" type="twist_rc.py" respawn="false" output="screen"/>
>>>>>>> 9d7518807e77ecbbdcd2e916172521b0ea15b235
</launch>
